### PR TITLE
remove generation of sap hana template

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
remove generation of sap hana template

We don't want to ship sap hana template in current release

**Special notes for your reviewer**:

**Release note**:
```
Remove sap hana template
```
Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: generate-templates.yaml [hunk 1]
```diff
@@ -40,20 +40,20 @@
...
-      - {flavor: tiny,  workload: saphana, default: False}
-    vars:
-      os: rhel8
-      icon: rhel
-      majorrelease: rhel8
-      oslabels: "{{ rhel8_labels }}"
-      osinfoname: "{{ oslabels[0] }}"
-      cloudusername: cloud-user
+#
+#  - name: Generate RHEL 8 SAP HANA template
```
Comment (rmohr) [inline generate-templates.yaml:44]: How about just adding something like

```
when: 0 > 1
```

then people can enable it easily if needed.

Comment (akrejcir) [reply generate-templates.yaml:44]: I will add it in another PR. Karel is on pto now, and I don't have permissions to edit this PR.

**Pull request merged**